parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -431,8 +431,9 @@ ensureAddons() {
   rm -Rf ${ADDONS_DIR}/init
   replaceAddonsInit
   {{/* Force re-load all addons because we have changed the source location for addon specs */}}
-  retrycmd 10 5 30 ${KUBECTL} delete pods -l app=kube-addon-manager -n kube-system || \
-  retrycmd 120 5 30 ${KUBECTL} delete pods -l app=kube-addon-manager -n kube-system --force --grace-period 0 || \
+  ADDON_POD_NAME=$(k get pods -l app=kube-addon-manager -n kube-system --no-headers -o custom-columns=":metadata.name")
```

Comment:
Since we see `No resources found` I think the pod hasn't been created yet.  We might be better to wait for the pod to start up then proceed with the delete.  I am thinking that `ADDON_POD_NAME` might be returning invalid data here if no resources are found?

None of the clusters started up...